### PR TITLE
Add crud

### PR DESCRIPTION
**Pull request by dkancalade**
- added post to listing endpoint
- added put to listing endpoint
- added delete to listing endpoint
- did basic tests for db helper functions and passed them
- passed basic test for delete endpoint
- still working on put and post tests

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -35,32 +35,70 @@ app.get('/listings', function (req, res) {
       res.sendStatus(500);
     } else {
       res.json(data);
```
Comment (davidlira1) [inline server/index.js:37]: for res.json(data), you can also send a status of success along with it

File: server/index.js [hunk 2]
```diff
@@ -35,32 +35,70 @@ app.get('/listings', function (req, res) {
...
+});
 
-app.get('/images', urlencodedParser, function (req, res) {
-  console.log('server /images');
+app.get('/images', function (req, res) {
   db.selectImages(req.body, function(err, data) {
     if (err) {
       res.sendStatus(500);
     } else {
       res.json(data);
```
Comment (davidlira1) [inline server/index.js:91]: this one too, you can send a status of success along with res.json(data). 
res.status(200).json(data)

**Review by davidlira1 — APPROVED** [review]
looks good Dustin!

File: server/index.js (hunk 1 above)
Comment (dkancalade) [reply server/index.js:37]: ty

**Pull request merged**